rplugin/python3/LanguageClient/LanguageClient.py

```diff
@@ -190,19 +190,46 @@ def start(self) -> None:
 
         self.defineSigns()
 
-        try:
-            self.nvim.call('cm#register_source', dict(
-                name='LanguageClient_%s' % languageId,
-                priority=9,
-                scopes=[languageId],
-                abbreviation='',
-                cm_refresh='LanguageClient_completionManager_refresh'))
-        except Exception as ex:
-            logger.warn("register completion manager source failed.")
-
         logger.info('End LanguageClientStart')
 
-        self.initialize([])
+        def cb(result: Dict):
```

Comment:
Can you make this a separate function and call it from self.handleInitializeResponse using self.nvim.async_call?